README.md

```diff
@@ -1,2 +1,30 @@
-# Project-setup-and-mobile-main
-Project setup and mobile skeleton
+# Portfolio-setup-and-mobile-first
+
+Description of the project:
+- Set up a new repository and prepare it for development using best practices (e.g. linters).
+- Build the first 2 sections of the mobile website using the template you chose in the previous step.
+
+Built with: Javascript, HTML and CSS
+
+General Requirements:
+- Make sure that there are no linter errors
+- Make sure that you used correct GitHub Flow
+- Make sure that you documented your work in a professional way
+
+Project Requirements:
+- For this project, you should use the HTML&CSS linter
+- The 2 sections you need to build for this project are:  the tool bar and the headline section
```

Comment:
- [Optional] Kindly follow [this](https://github.com/microverseinc/readme-template) to create your `README.md` file. This will make your work look more professional.